card/src/main/java/com/adyen/checkout/card/DualBrandedCardUtils.kt

```diff
@@ -0,0 +1,29 @@
+package com.adyen.checkout.card
+
+import com.adyen.checkout.card.data.CardType
+import com.adyen.checkout.card.data.DetectedCardType
+
+object DualBrandedCardUtils {
+
+    fun sortBrands(cards: List<DetectedCardType>): List<DetectedCardType> {
+        return when {
+            cards.size <= 1 -> cards
+            else -> {
+                val hasCarteBancaire = cards.any { it.cardType == CardType.CARTEBANCAIRE }
+                val hasVisa = cards.any { it.cardType == CardType.VISA }
+                val hasPlcc = cards.any {
+                    it.cardType == CardType.UNKNOWN &&
+                        (it.cardType.txVariant.contains("plcc") || it.cardType.txVariant.contains("cbcc"))
```

Comment:
Do we have these strings ("plcc" and "cbcc") somewhere already? Can we extract them if not?

Reply:
Yeah I thought we would let this logic to the backend anyway.

Reply:
For now I implemented it in the same way as Web does. When backend implements it we can remove it.